### PR TITLE
Set up e2e tests in dashboard-next

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
Set up e2e tests in dashboard-next

Issue: https://github.com/eclipse/che/issues/16910

### CODE REVIEW

**Comment by akurinnoy** [comment]
@olexii4 please take a look at this PR

File: tests/e2e/TestConstants.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export const TestConstants = {
```
Comment (olexii4) [inline tests/e2e/TestConstants.ts:13]: To be honest, I do not like the camel-case for a constant which is a simple object.

File: tests/e2e/driver/CheDashboardReporter.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+const screenCatcher: ScreenCatcher = testContainer.get(CLASSES.ScreenCatcher);
+let methodIndex: number = 0;
+let deleteScreencast: boolean = true;
+
+class CheDashboardReporter extends mocha.reporters.Spec {
+  constructor(runner: mocha.Runner, options: mocha.MochaOptions) {
+    super(runner, options);
+
+    runner.on('start', async (test: mocha.Test) => {
+//       const launchInformation: string =
```
Comment (olexii4) [inline tests/e2e/driver/CheDashboardReporter.ts:34]: Should we leave these lines like a comment?

**Review by olexii4 — APPROVED** [review]

File: tests/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# How to launch tests
+
+ - dashboard should be deployed
+
+ - set the ```TEST_BASE_URL``` env variable ```export TEST_BASE_URL=<dashboard url>```, 
+ if dashboard URL is ```http://localhost:3000``` this action may be scipped
+
```
Comment (olexii4) [inline tests/e2e/README.md:7]: It cannot login with CHE multiuser. It supports 'https://che.openshift.io' for now.

**Comment by olexii4** [comment]
Tested. Excellent job. Thanks

**Review by olexii4 — APPROVED** [review]

File: tests/e2e/README.md (hunk 3 above)
Comment (akurinnoy) [reply tests/e2e/README.md:7]: yes, it was done deliberately as the issue it just to setup e2e test environment. Next step will be be to mock all API requests and run e2e tests without interactions with backend.

**Pull request merged**